tests/dummy/app/pods/docs/advanced/switching-between-scenarios/template.md

```diff
@@ -0,0 +1,66 @@
+# Switching between scenarios
+
+You can use the scenarios directory to organize your scenario files. Scenarios
+might be useful if you need to conduct [user acceptance testing](https://en.wikipedia.org/wiki/Acceptance_testing#User_acceptance_testing)
+of your application using Mirage data. For example, you might test whether a user 
+can complete a set of tasks specific to unique circumstances defined in a scenario.
+
+Currently, scenarios are not a first-class API: Mirage only uses the default scenario
+in your development environment.
+
+That said, you can still configure your default scenario file to honor options passed
+to the runtime environment. For example, if you created a scenario file called `some-scenario.js`
+and wanted to load that scenario based on an environment variable called `MIRAGE_SCENARIO`, you
+could add the following to `scenarios/default.js`:
```

Comment:
Requested changes for first three paragraphs: generalize the motivation a bit (not just user acceptance testing), and open with the bit about not being a first-class API. Something like:

> Mirage adds a `/scenarios` directory to your project, where you'll find a `default.js` file. The function in this file will run in development mode and act as your "development seeds". In testing it's ignored, so that your Mirage server starts out with a clean database. You can then use each test to seed Mirage as needed for that test. In this way, each test acts as its own Mirage scenario.

> Originally Mirage was going to add first-class support for development-time scenario switching, but right only the the `scenarios/default.js` file is used by Mirage. However, you can use various strategies to configure the behavior of your default scenario, to support multiple development server states or to facilitate things like running [user acceptance testing](https://en.wikipedia.org/wiki/Acceptance_testing#User_acceptance_testing). For example, you might want to test whether a user can complete a set of tasks specific to unique circumstances defined in a scenario.

> One strategy you could take is to configure your default scenario file to honor options passed to the runtime environment. For example...